src/views/ProfileSettingsPage.vue

```diff
@@ -46,16 +47,16 @@ const saveProfile = () => {
 
 const onFileUploadToFirebase = () => {
   if (iconRef.value && iconRef.value.files) {
-    const file: File = iconRef.value.files[0]
-    icon.value = URL.createObjectURL(file)
+    // prefer-destructuring
+    uploadFile = iconRef.value.files[0] // eslint-disable-line
+    icon.value = URL.createObjectURL(uploadFile)
   }
 }
```

Comment:
@/stores/konva/image.tsのactionsにあるメソッドのように
イベントのコールバックの引数でEventが取得できるのでiconRefを使わずにinput要素とFileオブジェクトを取得できると思います。

```suggestion
const onFileUploadToFirebase = (e: Event) => {
    const inputElement = e.target as HTMLInputElement
         if (inputElement === null) return
         const { files } = inputElement
         if (files === null) return
         const file = files.item(0)
         if (file === null) return
         icon.value = URL.createObjectURL(uploadFile)
         inputElement.value = ''
}

// @/stores/konva/image.tsのメソッド
addImageList(e: Event) {
      const inputElement = e.target as HTMLInputElement
      if (inputElement === null) return
      const { files } = inputElement
      if (files === null) return
      const file = files.item(0)
      if (file === null) return
      this.loadImage(file)
      inputElement.value = ''
    },
```